README.md

```diff
@@ -44,11 +287,106 @@ import (
 }
 ```
 
+### Structure
+
+```
+.
+├── edgecast
+	package containing the main functionality for sdk.
+	Please add new client and model folders for new services here.
+│   ├── eclog
+		defines the the implementation and helper methods for logging
+│   ├──	internal
+		package containing helper methods and shared functionality used in sdk
+		please add new helper methods here
+│   │	├── collectionhelper
+			helper methods for working with aggregate/collection types
+│   │	├── jsonhelper		
+			helper methods for working with json
+│   │	├── testhelper
+			helper methods used in testing
+│   │	├── ecauth
+			authentication layer for oauth 2.0 and token based authentication
+│   │	├── ecclient
+			package client provides a base client implementation for interacting 
+			with edgecast cdn apis.
+			configuration and authentication types are also provided.
+│   ├── customer
+		client files for interacting with customer api
+		model files for customer
+│   ├── edgecnamee
+		client files for interacting with edge cname api
+		model files for edge cnamee
+│   ├── origin
+		client files for interacting with customer origin api
+		model files for customer origin
+│   ├── routedns
+		client files for interacting with route (dns)
+		model files for route (dns)
+│   ├── rtld
+		client files for interacting with real time log delivery api
+│   ├── rtldmodels
+		model files for real time log delivery
+│   ├── rulesengine
+		client files for interacting with rules engine api
+		model files for rules engine
+│   ├── waf
+		client files for interacting with waf api
+		model files for waf
+│   ├── shared
+		shared models and enums
+│   ├── config
+		defines the configuration of sdk services
+│   ├── doc
+		please add new docs here as needed
+│   ├── version
+		lists the latest version of sdk		
+├── example
+	example files to get started using the services
+├── template
+	template files used to generate client files and models using swagger api 
+	documentation
+├── Makefile
+        This Makefile should contain all testing and building operations.
+
+```
+
+### Contributing
+
+Please refer to the [contributing.md] (https://github.com/EdgeCast/ec-sdk-go/blob/main/Contributing.md) 
```

Comment:
For some reason, `[contributing.md]` isn't showing up as a link... maybe because of the space between ] and ( ?